### PR TITLE
allow multiple comment sections on a single page

### PR DESCRIPTION
**Pull request by Jamiras**

### CODE REVIEW

File: public/js/all.js [hunk 1]
```diff
@@ -533,23 +534,21 @@ $(function () {
...
 
-function removeComment(artID, commentID) {
+function removeComment(artTypeID, artID, commentID) {
   if (!window.confirm('Ary you sure you want to permanently delete this comment?')) {
     return false;
   }
 
   var posting = $.post('/request/comment/delete.php', { a: artID, c: commentID });
-  posting.done(onRemoveComment);
+  posting.done(function onRemoveComment(data) {
```
Comment (luchaos) [inline public/js/all.js:543]:
```suggestion
  posting.done(function(data) {
```

**Review by luchaos — APPROVED** [review]

File: public/js/all.js (hunk 1 above)
Comment (Jamiras) [reply public/js/all.js]: Was modelled after another recent change: https://github.com/RetroAchievements/RAWeb/pull/927/files#diff-f6d8800068b5d78d8a744d9f951adc97ee8738c9cda6636c3ccea921f059cce5R53

Corrections made.

Comment (luchaos) [reply public/js/all.js]: I missed that detail in the other PR - thanks 🙏

**Comment by MrOwnership** [comment]
Comments on pages that don't get reloaded on success (Hash, Leaderboard, Ticket) will just just be stuck on the loading gif after clicking the submit button.

https://github.com/RetroAchievements/RAWeb/blob/baa705c871d4a566e1c85c13aaf91d0854b5603f/public/js/all.js#L121-L133

NVIDIA_Share_Zaa9nq9FDN

**Pull request merged**